src/bootstrap-table.js

```diff
@@ -690,20 +690,18 @@ class BootstrapTable {
 
         if (buttonConfig.hasOwnProperty('attributes')) {
           for (const [attributeName, value] of Object.entries(buttonConfig.attributes)) {
-            buttonHtml += ` ${attributeName}="${Utils.calculateObjectValue(opts, value)}"`
+            buttonHtml += ` ${attributeName}="${value}"`
```

Comment:
If we merge this change, its no more possile to use functions!

Reply:
![image](https://user-images.githubusercontent.com/2117018/93714497-a844c700-fb95-11ea-96a9-e0f529554e08.png)

I don't think we need to support function here

Reply:
I think we should support it, because there are usecases where a static string is not enough (e.g. to use as text `user` or `users` based on some data).
So we should improve the documentation instead of remove this feature.